docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A fresh approach for date and time inputs
+description: Check out what's new with the next-gen pickers in earlier v6 versions.
+date: 2022-12-20T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'News']
+---
+
+There's a lot of exciting news in MUI X v6, but there's hardly anything comparable to the revamp we're delivering for the [Date and Time Pickers](https://next.mui.com/x/react-date-pickers/getting-started/).
+
+It's the result of our research and effort to improve these inputs' usability and overall customizability.
+We kept the best aspects of the previous versions, and we're fleshing out the package with several improvements and one fundamental change: a new input type to replace the masked text fields.
+
+## The new date and time fields
+
+This new input type we're talking about is a set of components shortly named [Fields](https://next.mui.com/x/react-date-pickers/fields/).
+They are the base for the new pickers and a revolution in terms of keyboard usability.
+
+In contrast with the masked inputs that only display simple text behind a formatting mask, and were often cumbersome, especially when editing dates, the new fields are smart and specialized in editing date and time numbers.
+It can recognize what's a month, day, or whatever piece of the date you are currently editing and act accordingly.
+
+For example, watch what happens when you edit the month with arrow keys.
+
+<video style="margin-bottom: 24px;" autoplay muted loop playsinline controls>
```

Comment:
The size of the video is off. It feels bigger than in real life. It feels better with:

```suggestion
<video style="margin-bottom: 24px; max-width: 615px" autoplay muted loop playsinline controls>
```

x the other videos to keep them 1:1 in scale.

Reply:
Since the video is relatively short, I thin that we could do
```suggestion
<video style="margin-bottom: 24px;" autoplay muted loop playsinline>
```

https://www.notion.so/mui-org/Blog-247ec2bff5fa46e799ef06a693c94917#cdbb73821e23409c94f184dc4f5d2e61

x the other relevant videos

Reply:
Do you know why there is this border on the recordings?

<img width="729" alt="Screenshot 2023-01-01 at 23 34 20" src="https://user-images.githubusercontent.com/3165635/210186444-f6ebed3a-9904-47f0-b9ab-765f7806fbbe.png">

Reply:
@oliviertassinari,
I'm re-recording the videos for this blog post and, meanwhile, trying to understand when the border happens. 
Initially, I thought it could be an artifact of a wrong aspect ratio or height set with an odd number, but none of those were confirmed correct; 

I couldn't get a consistent output without the border from a custom-sized screen recording with QuickTime, so now I'm removing the borders with an extra FFmpeg step to cut one pixel from the top and one from the bottom.

`ffmpeg -i masked-input.mp4 -filter:v "crop=in_w:in_h-2" -c:a copy masked-input_crop.mp4`